hi-ml-cpath/src/health_cpath/configs/classification/DeepSMILEPanda.py

```diff
@@ -70,7 +70,8 @@ def __init__(self, **kwargs: Any) -> None:
 
     def setup(self) -> None:
         BaseMILTiles.setup(self)
-        self.ssl_ckpt_run_id = innereye_ssl_checkpoint_binary
+        # If no SSL checkpoint is provided, use the default one
+        self.ssl_ckpt_run_id = self.ssl_ckpt_run_id or innereye_ssl_checkpoint_binary
```

Comment:
along the line of my comment above `innereye_ssl_checkpoint_binary` could be `innereye_panda_ssl_checkpoint_binary`

Reply:
I was trying to avoid having anything hardcoded since these run ids are workspace dependent, and we are using 2 so it would be a pain to change this variable in order to change workspace. IMO it is easier to specify it from the command line.
 To make it easier for us to remember which run id to specify on the command line, we could create variables in the Makefile instead?

Reply:
variables in the makefile sounds good to me